quickstart.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This file should be named `bootstrap.yml`

Reply:
done